openedx/core/djangoapps/user_authn/views/tests/test_logout.py

```diff
@@ -28,6 +28,8 @@ def setUp(self):
         super(LogoutTests, self).setUp()
         self.user = UserFactory()
         self.client.login(username=self.user.username, password='test')
+        self.saml_backend = 'tpa-saml'
+        self.learning_portal_root_url = 'http://localhost:8734/'
```

Comment:
`learning_portal_root_url` should be stored in settings and should use that setting both in implementation and test case.

Reply:
Added.